frontend/javascript/react/components/instagram_feed/instagram_feed.tsx

```diff
@@ -34,16 +35,27 @@ const InstagramFeed: FC = () => {
   }
   return (
     <div className="instagram-feed">
-      <div className="instagram-feed--grid">
-        {data.map((post) => (
-          <InstagramCard
-            key={post.id}
-            instagramPost={post}
-            classes="instagram-feed--grid--item"
-          />
-        ))}
-      </div>
+      <article>
+        <div className="instagram-feed-header">
+          <h5>1890bryantstudios</h5>
```

Comment:
Should this be @1890bryantstudios ?
```suggestion
          <h5>@1890bryantstudios</h5>
```

Reply:
yup, can do!